skins/contrib/Mozilla/global.css

```diff
@@ -337,7 +337,7 @@ hr {
 #bugzilla-body {
     background: none;
     border: none;
-    color: #404040;
+    color: #202020;
```

Comment:
Technically, we could just use color: #000 here, but if the goal is to *intentionally* lower the contrast of the text vs. the white background, then it makes sense that we're up in the #222/444 ranges.